feature/flags.go

```diff
@@ -46,3 +46,6 @@ const PersistentStorage = "persistent-storage"
 // StrictMigration will cause migration to error if there are unexported
 // values for annotations, status, status history, or settings.
 const StrictMigration = "strict-migration"
+
+// CAASModel enables creating models on CAAS infrastructure (k8s, etc)
+const CAASModel = "caas-model"
```

Comment:
Unless you had a particular reason for including "model" in the name could it just be `const CAAS = "caas"`?  `</bikeshedding>`